content/influxdb/cloud/get-started.md

```diff
@@ -48,27 +48,29 @@ The examples below use the [NOAA NDBC sample data](/influxdb/v2.0/reference/samp
 Use [sample data](/influxdb/cloud/reference/sample-data/) to quickly populate InfluxDB with sample time series data. Sample data sets delivered with Cloud are used throughout this documentation site to provide context, best practices, and clear examples for how InfluxDB Cloud features and functions work.
 
 1. Click **Data > Bucket** and click **Create Bucket** to create a bucket and name it **noaa**.
-2. Do one of the following to download sample data: 
-   - [Install community templates](#add-sample-data-with-community-templates) 
+2. Do one of the following to download sample data:
+   - [Install community templates](#add-sample-data-with-community-templates)
    - [Write sample data with an InfluxDB task](#write-sample-data-with-an-influxdb-task)
 
-### Add sample data with community templates 
+### Add sample data with community templates
 
-1. Visit the **InfluxDB templates page** in the InfluxDB Cloud UI. Click **Settings** > **Templates** in the navigation menu on the left.
+1. In the navigation menu on the left, select **Settings** > **Templates**.
 
     {{< nav-icon "settings" >}}
-      
-2. Paste the Sample Data community temple URL in **resource manifest file** field:
 
-    ```
+2. Paste the [Sample Data community template URL](https://github.com/influxdata/community-templates/blob/master/sample-data/sample-data.yml) in the **resource manifest file** field and click the **LOOKUP TEMPLATE** button.
```

Comment:
> There's a chance these all-caps buttons could be changed in the future. Using the `caps` shortcode rather than actual capital characters applies styles to match the UI and, if they ever do change, makes it really easy to update them all.

@sanderson Totally forgot - thanks